CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
       of returning `none` when the operation did not return output.
   - **CUMULUS-2309**
     - Removed `@cumulus/api/models/granule.unpublishAndDeleteGranule` in favor of `@cumulus/api/lib/granule-remove-from-cmr.unpublishGranule` and `@cumulus/api/lib/granule-delete.deleteGranuleAndFiles`.
+  - **CUMULUS-2385**
+    - Updated `sf-event-sqs-to-db-records` to write a granule's files to PostgreSQL only after the workflow has exited the `Running` status.
```

Comment:
We should probably explciitly caveat that this *also* updates the user workflow-includeable version of this lambda's behavior.

Reply:
That's a good point. What about something like this?

https://github.com/nasa/cumulus/pull/2168/commits/c1c76d01d2dc99d7ba9eaa04f53a0f7323148931